src/guide/4-initial-conditions/1-agent-populations.rst

```diff
@@ -1,3 +1,25 @@
 Agent Populations
 =================
-To initialise your model it is necessary to create the initial agent population and define the initial values of environmental properties. These can either be statically defined in a flat file, or dynamically generated.
\ No newline at end of file
+To initialise your model it is necessary to create the initial agent populations. These can either be statically defined in
+a flat file, or dynamically generated. This section deals with dynamically generating agents which is the recommended default
+method. For information on loading from files, please see **TODO: link to initialise from disk**
```

Comment:
which is the recommended default method, however if publishing results you may need to export the initial population, as random seeds cannot guarantee the same stream of random numbers if different compilers/versions are used.

Something to this effect should be clearly stated tbh.

Reply:
P.s. I'm not ignoring your feedback on these, just want to get the bulk of the actual documentation up so there is something complete and usable, and then I'll come back to address your points

Reply:
Yeah don't worry. I just have alot of shot gaps with my current Primage faff. So I can do context-free quick tasks like reviewing stuff, but it's alot harder for me to dig into working on agent ID when I'm likely to be interrupted in the short term.